### ISSUE TITLE
Version failed with json serializable

### ISSUE THREAD

**Issue by Katekko** [objectbox/objectbox-dart#211]
```BASH
Running "flutter pub get" in credito_app...                     
Because no versions of objectbox_generator match >0.13.0 <0.14.0 and objectbox_generator 0.13.0 depends on source_gen ^0.9.0, objectbox_generator ^0.13.0 requires source_gen ^0.9.0.


And because json_serializable >=4.1.0 depends on source_gen ^1.0.0, objectbox_generator ^0.13.0 is incompatible with json_serializable >=4.1.0.
So, because ekko depends on both json_serializable ^4.1.0 and objectbox_generator ^0.13.0, version solving failed.
pub get failed (1; So, because ekko depends on both json_serializable ^4.1.0 and objectbox_generator ^0.13.0, version solving failed.)
exit code 1
```

Pubspec:
![image](https://user-images.githubusercontent.com/13632735/112373643-f3b06d00-8cbf-11eb-8b09-76c24e7523c9.png)

Would be nice if upgrade the build too

```BASH
Running "flutter pub get" in credito_app...                     
Because no versions of objectbox_generator match >0.13.0 <0.14.0 and objectbox_generator 0.13.0 depends on build ^1.0.0, objectbox_generator ^0.13.0 requires build ^1.0.0.

And because json_serializable >=4.0.3 depends on build ^2.0.0, objectbox_generator ^0.13.0 is incompatible with json_serializable >=4.0.3.

So, because ekko depends on both json_serializable ^4.0.3 and objectbox_generator ^0.13.0, version solving failed.
pub get failed (1; So, because ekko depends on both json_serializable ^4.0.3 and objectbox_generator ^0.13.0, version solving failed.)
exit code 1

```

**Comment by vaind** [comment]
Fixed in 861f9b633c56ccc658eeaf35ae3d9b1ddcfd5b84 - if that works, I'll push before long (probably tomorrow)

**Comment by vaind** [comment]
pushed generator 0.13.0+1 with the dependency updates. Running `pub get` or `pub upgrade` should do it.
